File: codigos/1 - estrutura sequencial/1008 - Salario/Main.dart
import "dart:io";

main() {
  int number = int.parse(stdin.readLineSync()!);
  int horas = int.parse(stdin.readLineSync()!);
  double salario_hora = double.parse(stdin.readLineSync()!);

  String salario_total = (horas * salario_hora).toStringAsFixed(2);

  print("NUMBER = $number");
  print("SALARY = U\$ $salario_total");
}

File: codigos/1 - estrutura sequencial/1008 - Salario/Main.cs
using System; 

// class URI {
    // static void Main(string[] args) {
            int number = int.Parse(Console.ReadLine());
            int horas = int.Parse(Console.ReadLine());
            double salario_hora = double.Parse(Console.ReadLine());

            string salario_total = Math.Round(horas * salario_hora, 2).ToString("F2");

            Console.WriteLine($"NUMBER = {number}");
            Console.WriteLine($"SALARY = U$ {salario_total}");
    // }
// }